beamer/state_machine.py

```diff
@@ -317,11 +323,8 @@ def _handle_claim_made(event: ClaimMade, context: Context) -> HandlerResult:
 
 def _handle_claim_stake_withdrawn(event: ClaimStakeWithdrawn, context: Context) -> HandlerResult:
     claim = context.claims.get(event.claim_id)
-
-    # Check if claim exists, it could happen that we ignored the request because of an
-    # invalid token pair, and therefore also did not create the claim
     if claim is None:
-        return True, None
+        return False, None
```

Comment:
Eh, this can potentially result in a big number of events accumulating over time in an agent that does not support the token pairs involved. I'm not sure how to address that.

Reply:
That's true. I haven't better solution though.

Reply:
I'm working on a solution for this. Please do not merge yet.